### DISCUSSION TITLE
LaTeX editing slowed down when treesitter is enabled

### DISCUSSION THREAD

**Discussion by Silzinc** [nvim-treesitter/nvim-treesitter#7214] [Q&A]
Hello ! I am using LazyVim with its [default treesitter config](https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/plugins/treesitter.lua) + the `latex` parser installed.

When I edit a TeX file, there is often a latency when inserting a character. This only happens when treesitter is enabled : running `lua vim.treesitter.stop()` makes it smooth again.

Note that I am using VimTeX, but disabling it (as well as removing texlab) does not solve the problem. This can be observed in a simple file : 

```latex
\documentclass{article}
```

Just go at the last char, enter insert mode and insert a space. There is always a latency at this moment. Is this a known issue or do I have a bad config somewhere ? Thank you in advance.

**Comment by clason** [comment]
If you use VimTeX, don't install the latex parser (or don't enable it for highlighting).

**Reply by Silzinc** [reply]
Well... that's rather disappointing. I liked how treesitter highlighted my code while VimTeX does not change anything to the default support (in what my config outputs at least).

Is there any way to tell VimTeX not to try to highlight the text ? Or even just a replacement plugin that allows this ?
